tests/test_data.sql

```diff
@@ -1,8 +1,9 @@
-insert into story values (1, 'test-story');
-insert into node values (1, 1, 'Root', 'root');
-insert into node values (2, 1, 'Root1', 'root1');
+insert into story values (1, 'test_story');
```

Comment:
Can store as a JSON and use SQLAlchemy to load the date to DB.

Reply:
I think that SQLAlchemy can`t parse json properly. I suggest to create this data using SQLAlchemy models

Reply:
Right, but you can and use `db.session.insert`